livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/beta/assistant_llm.py

```diff
@@ -95,7 +103,13 @@ def __init__(
         self._assistant_opts = assistant_opts
         self._running_fncs: MutableSet[asyncio.Task[Any]] = set()
 
-        self._sync_openai_task = asyncio.create_task(self._sync_openai())
+        self._sync_openai_task: asyncio.Task[AssistantLoadOptions] | None = None
+        try:
+            self._sync_openai_task = asyncio.create_task(self._sync_openai())
+        except Exception:
+            logger.error(
+                "failed to create sync openai task. This can happen when instantiating without a running asyncio event loop (such has when running tests) will try lazily later"
```

Comment:
```suggestion
                "failed to create sync openai task. This can happen when instantiating without a running asyncio event loop (such has when running tests)"
```